index.html

```diff
@@ -36,8 +38,8 @@
 
       // if there is a previously published draft, uncomment this and set its YYYY-MM-DD date
       // and its maturity status
-      previousPublishDate:  "2021-08-26",
-      previousMaturity:  "FPWD",
+      previousPublishDate: "2021-08-26",
+      previousMaturity: "FPWD",
```

Comment:
The `previousPublishdDate` and `previousMaturity` must be commented out. At this moment, for the FPWD, there is no such previous publication and the result gets a header entry that pubrules rejects (for good reasons...)